test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests.cs

```diff
@@ -174,29 +174,7 @@ public SamplesEndToEndTests(TestFixture fixture)
             Assert.Equal("Testing", TestHelpers.RemoveByteOrderMarkAndWhitespace(result));
         }
 
-        [Fact(Skip = "Needs investigation")]
-        public async Task HttpTrigger_CSharp_Poco_Post_Xml_Succeeds()
```

Comment:
Decided to just remove the xml goo from the sample. I could have kept it and enabled the test, but I don't think we need it.

Reply:
Is this scenario covered elsewhere? Or do we get it automatically now?

Reply:
This test was really just testing platform xml handling, not our code. I just added it long ago to verify it worked. However it made the sample grubby with all the DA xml annotations, so removing. Test not really needed.